### PR TITLE
PEPPER-482 . add consent-assent MR pdf config v3

### PR DESCRIPTION
**Pull request by ssettipalli**
## Context

add missed consent-assent MR pdf config v3

### CODE REVIEW

File: pepper-apis/studybuilder-cli/studies/osteo/patches/osteo-v3-medicalrecords-pdfs.conf [hunk 1]
```diff
@@ -133,6 +133,75 @@
           ]
         }
       ]
+    },
+
+    {
+      "name": "osproject-release-consent-assent",
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/patches/osteo-v3-medicalrecords-pdfs.conf:139]: PDF config needed for consent_assent 7+ age minors

File: pepper-apis/studybuilder-cli/studies/osteo/patches/osteo-v3-medicalrecords-pdfs.conf [hunk 2]
```diff
@@ -133,6 +133,75 @@
...
+              "type": "PARTICIPANT"
+            },
+            {
+              "type": "ACTIVITY",
+              "activityCode": "RELEASE_MINOR",
+              "versionTag": "v3"
+            },
+            {
+              "type": "ACTIVITY",
+              "activityCode": "CONSENT_ASSENT",
```
Comment (ssettipalli) [inline pepper-apis/studybuilder-cli/studies/osteo/patches/osteo-v3-medicalrecords-pdfs.conf:158]: 7+ AGE minors activity CONSENT_ASSENT

**Review by otchet-broad — APPROVED** [review]
Good catch!

**Review by otchet-broad — APPROVED** [review]
Good catch!

**Comment by ssettipalli** [comment]
Applied patch to Dev.
Signed up CY81W3NDECEFTL2A5IRO and verified that pdf can be generated.

**Pull request merged**